models/image.py

```diff
@@ -877,23 +674,35 @@ def from_images(cls, images):
         output.exp_time = sum([image.exp_time for image in images])
 
         # start MJD and end MJD
-        output.mjd = min([image.mjd for image in images])
-        output.end_mjd = max([image.end_mjd for image in images])
+        output.mjd = images[0].mjd  # assume sorted by start of exposures
+        output.end_mjd = max([image.end_mjd for image in images])  # exposure ends are not necessarily sorted
```

Comment:
If the exposure starts are sorted, then the exposure ends should be sorted too.  The only way for them to mismatch is if exposure B started while exposure A was still going, and was a short exposure that finished first.  But that can't happen of all of the images are from the same instrument.

We should add to the docstring that the images need to be sorted by mjd if the code is going to assume that.

Reply:
I think there's very little cost to not assuming that the images don't overlap each other (as per our discussion of getting reference images from different instruments), so I will leave it here. 

The images are sorted by `mjd` a few lines above that.